spokestack/vad/webrtc.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module contains the webrtc component for
+voice activity detection (vad)
+"""
+
+import webrtcvad  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+MODE = {"quality": 0, "low_bitrate": 1, "aggresive": 2, "very_aggresive": 3}
+
+
+class VoiceActivityDetector:
+    """ This class detects the presence of voice in a frame of audio.
+
+    :param sample_rate: sample rate of the audio
```

Comment:
it's good to specify the units for values like this, like "sample rate of the audio, in Hz"
ditto for frame_width, vad_rise_delay, vad_fall_delay (milliseconds)

Reply:
that makes so much sense. thanks for pointing that out.